components/config/histogram_config.py

```diff
@@ -18,6 +19,8 @@ def get_component(columns, min_date, max_date, default_end_date):
 
     return html.Div([
         date_picker.get_component(min_date, max_date, default_end_date),
-        histogram_column_picker.get_component(columns),
+        axis_picker.get_component(columns, default_column, is_histogram=True),
+        html.Div(className='label', children='Maximum number of bins'),
```

Comment:
Using the class `config-label` here would ensure consistency across the different views.